app/controllers/encode_records_controller.rb

```diff
@@ -75,11 +75,11 @@ def paged_index
         [
           "<span data-encode-id=\"#{encode.id}\" class=\"encode-status\">#{encode_presenter.status}</span>",
           view_context.link_to(encode_presenter.id, Rails.application.routes.url_helpers.encode_record_path(encode)),
-          "<progress value=\"#{encode_presenter.progress}\" max=\"100\" data-encode-id=\"#{encode.id}\" class=\"encode-progress\"></progress>",
+          "<progress value=\"#{encode_presenter.progress}\" max=\"100\" data-encode-id=\"#{encode.id}\" class=\"encode-progress #{encode_presenter.status.downcase}\"></progress>",
           encode_presenter.display_title,
           view_context.link_to(encode_presenter.master_file_id, encode_presenter.master_file_url),
           view_context.link_to(encode_presenter.media_object_id, encode_presenter.media_object_url),
-          encode_presenter.created_at
+          encode_presenter.created_at.utc.strftime('%D %r')
```

Comment:
I have a concern about changing the date format because date searches might not match search results as they appear here.

Reply:
The search works for dates, but only in the `yyyy-mm-dd` format, which is the original format. Shall we change the format to that and keep the date formatting code here?

Reply:
The storage format is determined by something other than us, afaik (ActiveRecord or ActiveFedora, I think). I don't think there is a way for us to change that.

Reply:
Sorry I didn't mean that we need to change the stored format, but only the format shown in the UI. If we change this line to,
```
encode_presenter.created_at.strftime('%Y-%m-%d  %H:%M:%S')
```
the dates are shown as the same format as storage format. That way when a user enters the date into the search they are not confused. What do you think?

Reply:
That sounds good.